packages/mask/src/plugins/Collectible/base.ts

```diff
@@ -17,6 +17,7 @@ export const base: Plugin.Shared.Definition = {
             /opensea.io\/assets\/(0x[\dA-Fa-f]{40})\/(\d+)/,
             /rarible.com\/token\/(0x[\dA-Fa-f]{40}):(\d+)/,
             /zora.co\/collections\/(0x[\dA-Fa-f]{40})\/\d+$/,
+            /magiceden.io\/item-details\/([1-9A-HJ-NP-Za-km-z]{32,44})/,
```

Comment:
Any document for this ID schema?

Reply:
appears in the solana docs https://docs.solana.com/integrations/exchange#basic-verfication

Reply:
I see, thanks a lot.

Reply:
```suggestion
            // see https://docs.solana.com/integrations/exchange#basic-verfication
            /magiceden.io\/item-details\/([1-9A-HJ-NP-Za-km-z]{32,44})/,
```